### PR TITLE
[Gen ai] Log level path instead of level id on Amplitude/Statsig

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR logs the level path instead of level id when sending an event to Amplitude/Statsig. This was requested by @samantha-code .

Screenshot 2024-05-07 at 4 14 51 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/aichat/redux/aichatRedux.ts [hunk 1]
```diff
@@ -165,12 +166,25 @@ export const saveModelCard = createAsyncThunk(
...
       savedAiCustomizations,
       EVENTS.SAVE_MODEL_CARD_INFO,
-      levelId,
+      levelPath,
       dispatch
     );
   }
 );
 
+const getLevelPath = (progress: ProgressState) => {
```
Comment (bencodeorg) [inline apps/src/aichat/redux/aichatRedux.ts]: Is there a reason not to just grab the URL (something like `window.location.pathname`)?

Comment (fisher-alice) [reply apps/src/aichat/redux/aichatRedux.ts]: Oh cool! I don't think so. I'll go ahead and use `window.location.pathname` instead. Thanks!

Comment (fisher-alice) [reply apps/src/aichat/redux/aichatRedux.ts]: FYI - when I added level url to Dance AI analytics, I added `currentScriptLevelUrl`.  https://github.com/code-dot-org/code-dot-org/pull/55531
But now I see I could've used `window.location.pathname` instead. Wonder if I should remove this seemingly unnecessary property.

Comment (bencodeorg) [reply apps/src/aichat/redux/aichatRedux.ts]: Yeah that seems redundant to me, but I could be missing something :)

**Review by bencodeorg — APPROVED** [review]

**Pull request merged**